src/utilities/PlaneAveraging.cpp

```diff
@@ -100,7 +100,11 @@ void PlaneAveraging::plot_line_binary(std::string filename, int step, Real time)
 }
 
 template<typename IndexSelector>
-void PlaneAveraging::fill_line(const IndexSelector &idxOp, const amrex::MultiFab& velocity,  const amrex::MultiFab& tracer)
+void PlaneAveraging::fill_line(const IndexSelector &idxOp,
+                               const amrex::MultiFab& density,
+                               const amrex::MultiFab& velocity,
+                               const amrex::MultiFab& mueff,
+                               const amrex::MultiFab& temperature)
```

Comment:
I think it will be useful to split off `PlaneAveraging` utility into two parts: 1. that handles velocity and its first/second moments, and 2. a new class that can take any scalar and compute its averages, first/second moment... not just tempertaure.

Reply:
sounds good, I'll put it on the todo list